src/types/api/submission.ts

```diff
@@ -6,3 +8,21 @@ export type SubmissionResponseDto = {
 }
 
 export type SubmissionErrorDto = ErrorDto & { spcpSubmissionFailure?: true }
+
+export type SubmissionCountQueryDto = {
+  formId: string
+  startDate?: Date
+  endDate?: Date
+}
+
+export type SubmissionMetadataQueryDto = {
+  formId: string
+} & RequireAtLeastOne<{
```

Comment:
Just wanted to check, why don't we use `RequireExactlyOne` instead? Since it seems like we only use either `submissionId` or `pageNum`.

Reply:
you're right! i set to `RequireAtLeastOne` because the FE callsite passed in both parameters and i didn't want to change it yet so i made it be more permissive. but because our BE handler uses `RequireExactlyOne`, it still works the same way!

Reply:
like what i said above, can split into two separate functions to remove this ambiguity